config/routes.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
   namespace :api do
-    resources :articles, only: %i[index show]
+    resources :articles, only: %i[index show create]
```

Comment:
I think we have to change it to ```only: [:index, :show, :create]```. this works but the other way looks more readable.

Reply:
Fixed!